.gitignore

```diff
@@ -28,3 +28,8 @@ rspec.xml
 .vendor
 integration_run
 .mvn/
+qa/.ssh_config
+qa/.vagrant
+qa/.rspec
+qa/acceptance/.vagrant
+qa/Gemfile.lock
```

Comment:
we could, but not sure if makes sense! I know vendor is common directory, but not sure to add it as anyone could use different directories.